<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallingClientsUIItem</class>
 <widget class="QWidget" name="CallingClientsUIItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QLabel" name="label_Dial">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_Receive">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_CallingStatus">
     <property name="text">
      <string>未接通</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_CallingTime">
     <property name="text">
      <string>00:00</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_Play">
     <property name="text">
      <string>备用</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_Stop">
     <property name="text">
      <string>强制中断</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
